### PR TITLE
Use logs for deployment information in Jenkins

### PR DESCRIPTION
**Pull request by cescgina**
Instead of relying on the builds artifacts, get the deployment
information from the logs of the job. This allows to query for
information like the containers used in each openstck node, the services
or packages installed.

### CODE REVIEW

File: cibyl/sources/jenkins.py [hunk 1]
```diff
@@ -167,7 +171,8 @@ def __init__(self, url: str, username: str = None, token: str = None,
 
     @safe_request
     def send_request(self, query, timeout=None, item="",
-                     api_entrypoint="/api/json", raw_response=False):
+                     api_entrypoint="/api/json", raw_response=False,
+                     url_to_use=None):
```
Comment (bregman-arie) [inline cibyl/sources/jenkins.py:175]: How about calling it simply "url"? I understand that generate_query_url uses it, but that's a different namespace

File: cibyl/sources/jenkins.py [hunk 2]
```diff
@@ -499,90 +510,228 @@ def get_deployment(self, **kwargs):
...
-                                             raw_response=True)
-                break
-            except JenkinsError:
-                LOG.debug("Found no artifact %s for job %s", artifact_path,
-                          job_name)
-                continue
-
-        if artifact is None:
+        build_description = job["lastCompletedBuild"].get("description")
+        if build_description is None:
```
Comment (bregman-arie) [inline cibyl/sources/jenkins.py:521]: `if not build_description:` will catch also empty descriptions

Comment (cescgina) [reply cibyl/sources/jenkins.py]: Good catch, thanks

File: cibyl/sources/jenkins.py (hunk 1 above)
Comment (cescgina) [reply cibyl/sources/jenkins.py]: Makes sense, I changed it

**Review by bregman-arie — APPROVED** [review]
Really good stuff :)

**Pull request merged**